index.html

```diff
@@ -5,7 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <title>Dozer | Managed BOSH</title>
+    <title>Dozer | The GUI for BOSH</title>
+    <meta name="description" content="Dozer offers a graphical user interface (GUI) for BOSH that makes it easy to connect, monitor and manage multiple instances of BOSH." />
```

Comment:
Distinguishes from OpsManager which is essentially also a GUI for BOSH however it tries to hide BOSH while we try to make it easier to use plain BOSH (so maybe OS BOSH is not the right term).